src/main/java/object/BonusHolder.java

```diff
@@ -0,0 +1,9 @@
+package object;
+
+import bonus.Bonus;
+
+interface BonusHolder {
+
+    Bonus getBonus();
+    boolean isBonus();
```

Comment:
what is isBonus API method for? I expect that BonusHolder interface always behave itself as bonus

Reply:
`BonusHolder` is an interface of `GameObject` that could retrrieve `Bonus` itself. Bonus isn't `GameObject`, it's just the class that can apply bonus to `GameObject`